middleware/session/session_test.go

```diff
@@ -523,10 +524,6 @@ func Test_Session_Reset(t *testing.T) {
 		// Check that the session id is not in the header or cookie anymore
 		utils.AssertEqual(t, "", string(ctx.Response().Header.Peek(store.sessionName)))
 		utils.AssertEqual(t, "", string(ctx.Request().Header.Peek(store.sessionName)))
-
-		// But the new session id should be in the header or cookie
-		utils.AssertEqual(t, acquiredSession.ID(), string(ctx.Response().Header.Peek(store.sessionName)))
```

Comment:
`acquiredSession.ID()` is always the empty string after calling `acquiredSession.Save()`, so these asserts are redundant to the previous two asserts.